ironic/drivers/modules/drac/management.py

```diff
@@ -84,6 +88,14 @@
 # Clean steps constant
 _CLEAR_JOBS_CLEAN_STEPS = ['clear_job_queue', 'known_good_state']
 
+# System Management Constant
+_SERVICE_ROOT = '/redfish/v1/Managers/'
```

Comment:
I think _SERVICE_ROOT can be remove.  Doesn't look like it's used.

Reply:
Yes, It is unused constant value, Removed it.